test/e2e-contour-tests.sh

```diff
@@ -85,6 +85,59 @@ spec:
           logCmdOutput: "true"
 EOF
 
+cat <<EOF | kubectl apply -f -
+apiVersion: flagger.app/v1beta1
+kind: Canary
+metadata:
+  name: podinfo-service
+  namespace: test
+spec:
+  targetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: podinfo-service
+  progressDeadlineSeconds: 60
+  service:
+    port: 9898
+    portDiscovery: true
+    apex:
+      annotations:
+        test: "annotations-test"
+      labels:
+        test: "labels-test"
+    headers:
+      request:
+        add:
+          x-envoy-upstream-rq-timeout-ms: "15000"
+          x-envoy-max-retries: "10"
+          x-envoy-retry-on: "gateway-error,connect-failure,refused-stream"
+  analysis:
```

Comment:
Can you please remove the apex settings as they don't apply to existing services also remove analysis metrics and webhooks as they are not used in the tests. Please do this for all test scripts.

Reply:
I believe I addressed this feedback with my latest commit. Thank you for the feedback. 👍